src/parser/scanner.lex

```diff
@@ -36,6 +36,7 @@ IS_EMPTY                    (IS{blanks}EMPTY)
 IS_NOT_EMPTY                (IS{blanks}NOT{blanks}EMPTY)
 
 LABEL                       ([a-zA-Z][_a-zA-Z0-9]*)
+EXTEND_LABEL                ([^`]+)
```

Comment:
Would this lead to that users could write out any forms of identifiers, like `***`, `_____vid`？

Reply:
It's allowed, maybe we should check that property name shouldn't be the reserved name when create schema.